### PR TITLE
Network error toast

### PR DESCRIPTION
**Pull request by mjoyce91**
To recreate:
- Shut down mock-fsbid
- Access pages on TalentMAP
- Toast should appear at most every 10 seconds

Screen Shot 2020-10-08 at 2 23 10 PM

### CODE REVIEW

File: src/Containers/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -1,24 +1,22 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { assignmentFetchData } from 'actions/assignment';
```
Comment (elizabeth-jimenez) [inline src/Containers/Dashboard/Dashboard.jsx:4]: 🧼

File: src/api.js [hunk 2]
```diff
@@ -31,6 +32,21 @@ const debouncedLogout = throttle(
   { leading: true, trailing: false },
 );
 
+const debouncedNetworkAlert = throttle(
+  // eslint-disable-next-line global-require
+  // () => require('./store').store.dispatch(logoutRequest()),
```
Comment (elizabeth-jimenez) [inline src/api.js:37]: needed?

**Review by elizabeth-jimenez — APPROVED** [review]
Just some comments that may be left over from dev. Looks really good! 🍥 🍰 🥮 👍

**Comment by elizabeth-jimenez** [comment]
If it's a low LOE, checkout the resolutions for ipad Pro(out of view) and ipad(half out of view).

**Comment by mjoyce91** [comment]
@elizabeth-jimenez I was able to see it fine when emulating both of those devices

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez I was able to see it fine when emulating both of those devices

@mjoyce91 Dang, already moved on to another PR, should've taken a SS; I trust you!

**Pull request merged**